README.md

```diff
@@ -120,15 +120,29 @@ Compiling
 
 Compiling requires using CMake 3.1 or newer. GCC and Clang are both known to work to compile mGBA, but Visual Studio 2013 and older are known not to work. Support for Visual Studio 2015 and newer is coming soon.
 
-#### Docker building
+## Docker building
 
-The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms.
+The recommended way to build for most platforms is to use Docker. Several Docker images are provided that contain the requisite toolchain and dependencies for building mGBA across several platforms. 
+
+First, you need to make sure you are within the mGBA repository root directory, and it is located in a directory where it's in a *nix path. Docker is known to be stringent on this restriction. (See issue #1985 for details.) 
```

Comment:
This restriction is specific to Docker for WIndows only. Also, for linking to issues (I should double check the issue here specifically), use `[#XXXX](https://mgba.io/i/XXXX)` format to make sure it links properly.

Reply:
Oh this is cool! I'll update it. Thanks.